.changeset/old-games-teach.md

```diff
@@ -0,0 +1,5 @@
+---
+"@meso-network/meso-js": patch
+---
+
+explicitly set origin referrer policy on iframe
```

Comment:
```suggestion
Explicitly set [referrerPolicy](https://developer.mozilla.org/en-US/docs/Web/API/HTMLIFrameElement/referrerPolicy) on Meso iframes. This prevents a bug in Firefox browsers where frames cannot establish their parent's origin using `document.referrer` if the parent has a no-referrer policy set.
```

I think our changelogs should be a more descriptive. Mine may be a bit too wordy, but you get the idea.